backend/telemetry_core/src/state/chain_stats.rs

```diff
@@ -96,6 +96,27 @@ fn bucket_memory(memory: u64) -> (u32, Option<u32>) {
         48,
         56,
         64,
+        128,
+    }
+}
+
+fn kernel_version_number(version: &Box<str>) -> &str {
+    &version[0..version.find('-').unwrap_or(version.len())]
```

Comment:
"1.20.0+c144a98" is also a valid semver version but probably a weird edge not sure if's worth the effort to strip that one as well

linux kernel is probably not using it anyway but yeah technically possible

Reply:
so you could just use semver::Version here :)

Reply:
By `semver::Version` you mean using another crate for this?

For now I've manually parsed the kernel version to take into account the plus sign :D